test/e2e/cypress/integration/backups.spec.ts

```diff
@@ -0,0 +1,23 @@
+describe('Backups view', () => {
+  beforeEach(() => {
+    cy.clearLocalStorage();
+    cy.login();
+    cy.intercept('GET', '/api/backup/deviceAsFolder', { fixture: '/api/backups/deviceAsFolder.json' });
+    cy.intercept('GET', '/api/storage/v2/folder/130', { fixture: '/api/backups/storagev2folder130.json' });
+    cy.intercept('GET', '/api/storage/v2/folder/131', { fixture: '/api/backups/storagev2folder131.json' });
```

Comment:
I like this approach, and given in drive-web you cannot create backups, its good enough to fulfill the task.

Although it occurs to me to test the functionality completely we might be able create some backups via pure HTTP requests and then test them without intercepting? 🤔 

(correct me if it's not possible)

Reply:
It's doable but I would rather (for the moment) add an e2e test to drive server to test his functionality. 
This will help us to achieve test independence from each other since to upload a backup, first we need to create a device, and to create a device we need a user. So we are going to rely on the auth test to have a user.